tests/helpers.py

```diff
@@ -0,0 +1,16 @@
+import pathlib
+
+import awkward as ak
+
+from dask_awkward.io import from_json
+
+this_path = pathlib.Path(__file__).parent.resolve()
```

Comment:
Since I'm old fashioned, I wonder whether this gets you anything over
```
this_path = os.path.dirname(__fle__)
records_file = os.path.join(this_path, "data", "records.json")
```

Reply:
I personally find the `pathlib` variant more readable :)